### PR TITLE
task(26995282): fix pylint issues for 5kyu folder & use new test template

### PR DESCRIPTION
**Pull request by vmred**

### CODE REVIEW

**Review by red-redalert — APPROVED** [review]

**Review by red-redalert — CHANGES_REQUESTED** [review]
need to fix black errors

**Comment by vmred** [comment]
> need to fix black errors

done

File: katas/5kyu/Vector class #1/solution.py [hunk 1]
```diff
@@ -21,11 +21,11 @@
 class Vector:
     def __init__(self, *args):
         self.vector = list(args[0]) if len(args) == 1 else list(args)
-        self.x, self.y, self.z = self.vector
+        self.x, self.y, *self.z = self.vector
```
Comment (red-redalert) [inline katas/5kyu/Vector class #1/solution.py:24]: tests would fail here as `*self.z` is an array here

**Review by red-redalert — CHANGES_REQUESTED** [review]

File: katas/5kyu/Phone directory/solution.py [hunk 2]
```diff
@@ -33,16 +33,18 @@
...
         if "+" + num in line:
-            name = sub(".*<(.*)>.*", "\g<1>", line)
-            line = sub("<" + name + ">|\+" + num, "", line)
+            name = sub(".*<(.*)>.*", r"\g<1>", line)
+            line = sub(fr"<{name}>|\+{num}", "", line)
             address = " ".join(sub("[^a-zA-Z0-9.-]", " ", line).split())
-            return "Phone => %s, Name => %s, Address => %s" % (num, name, address)
+            return f"Phone => {num}, Name => {name}, Address => {address}"
+
+    return ''
```
Comment (red-redalert) [inline katas/5kyu/Phone directory/solution.py:50]: add `return ''` in the end of function to avoid pylint inconsistent return error

**Review by red-redalert — CHANGES_REQUESTED** [review]

File: katas/5kyu/Vector class #1/solution.py (hunk 1 above)
Comment (vmred) [reply katas/5kyu/Vector class #1/solution.py]: done

**Review by red-redalert — APPROVED** [review]

**Pull request merged**